src/Twilio.TaskRouter/Model/Filter.cs

```diff
@@ -17,6 +17,12 @@ public Filter()
         [DataMember(EmitDefaultValue = false, Name="friendly_name")]
         public string FriendlyName { get; set; }
 
+        [DataMember(EmitDefaultValue = false, Name = "filter_friendly_name")]
+        public string FilterFriendlyName {
+            get { return null; }

```

Comment:
If you don't expose a getter you'll get: System.Runtime.Serialization.InvalidDataContractException : No get method for property 'FilterFriendlyName' in type 'Twilio.TaskRouter.Filter'.

This is because we've marked FilterFriendlyName as a DataMember of our DataContract so that we can deserialize the "filter_friendly_name" field. And a valid data contract requires that all data members are able to be serialized, which means we must have a getter. 
c.f. >  https://msdn.microsoft.com/en-us/library/ms733127(v=vs.110).aspx

So to prevent "filter_friendly_name" from getting serialized we're returning a null on the getter since if a Data Member is set to the default value, then the field will not get serialized. 
c.f.  > https://msdn.microsoft.com/en-us/library/aa347792(v=vs.110).aspx

I previously looked into serialization callbacks, but the thing about using the OnSerializedAttribute is that it occurs after serialization has occurred, which means "filter_friendly_name" would still get serialized if we do return this.FriendlyName on the getter.